Dockerfile

```diff
@@ -13,7 +13,7 @@ RUN chmod 0644 /etc/cron.d/check-for-data
 # DOWNLOAD DATASET
 COPY ./download_dataset.sh /download_dataset.sh
 RUN chmod +x /download_dataset.sh
-RUN source /download_dataset.sh
+RUN /download_dataset.sh
```

Comment:
I think this will not work as `download_dataset.sh` sets an environnement variable which will not work if we do not `source` the file ? 
See : https://stackoverflow.com/questions/496702/can-a-shell-script-set-environment-variables-of-the-calling-shell

Reply:
Ok, but maybe we need to find another command because it failed on github actions for some reasons

Reply:
i retried a commit on develop and here is what I get

https://github.com/ambanum/OpenTermsArchive-api/runs/2022094682?check_suite_focus=true

Any idea?

Reply:
Basically Docker thinks that `source` is a file to run and tries to run it but fails because it can't find a script named `source`.
According to [this](https://stackoverflow.com/questions/20635472/using-the-run-instruction-in-a-dockerfile-with-source-does-not-work) the solution is :

`RUN /bin/bash -c "source /download_dataset.sh"`

I'll change this tomorrow :)